test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -97,9 +97,9 @@ Describe "Enter-PSHostProcess tests" -Tag Feature {
         }
 
         It "Can enter, exit, and re-enter another Windows PowerShell PSHost" -Skip:(!$IsWindows) {
-            # Start a Windows PowerShell job where the first thing it does is return $pid. After that, spin forever.
+            # Start a PowerShell job where the first thing it does is return $pid. After that, spin forever.
             # We will use this job as the target process for Enter-PSHostProcess
-            $powershellJob = Start-Job -PSVersion 5.1 {
```

Comment:
Need to document this breaking change

Reply:
Updated the doc issue: https://github.com/MicrosoftDocs/PowerShell-Docs/issues/5092